wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -209,7 +236,7 @@ fun parseType(field: DescriptorProtos.FieldDescriptorProto): String {
     }
     // TODO: Figure out group types
     DescriptorProtos.FieldDescriptorProto.Type.TYPE_GROUP -> ""
-    else -> TODO("else case found for ${field.type}")
+    else -> TODO("else case found for $ {field.type}")
```

Comment:
Is this extra space intentional? Does interpolation not require the `{` to be adjacent to the `$`?

Reply:
Not intentional...ooops